ui/components/component-library/tag-url/tag-url.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import { BaseAvatar } from '../base-avatar';
+import Box from '../../ui/box/box';
+import { Text } from '../text';
+import {
+  ALIGN_ITEMS,
+  COLORS,
+  DISPLAY,
+  JUSTIFY_CONTENT,
+  BORDER_COLORS,
+  SIZES,
+} from '../../../helpers/constants/design-system';
+
+export const TagUrl = ({ label, className, imageSource, cta, ...props }) => {
```

Comment:
We may also have a right icon so it could be worth having `leftComponent` and `rightComponent` props?

```
{leftComponent ? leftComponent : <AvatarAccount  {...avatarAccountProps} />}
<Text {...labelProps}>{label}</Text>
{rightComponent}
```
Or something to that effect. I would check with @Akatori-Design

Reply:
@georgewrmarshall  I am yet to work on the `rightComponent` , I had not added it yet. But the prop `cta` is for that only (I named it cta to keep it consistent with the mobile). I am not sure if we need `leftComponent ` prop specifically for the AvatarAccount, as per the figma. But yeah, like you mentioned it will be good to have @Akatori-Design thoughts on it.